africanlii/static/stylesheets/africanlii.scss

```diff
@@ -31,3 +31,7 @@ iframe.video {
     display: block;
   }
 }
+
+a:visited {
```

Comment:
is this necessary if peachjam already has it?

Reply:
fixed, I also had to set the button color for africanlii to black like we do in other liis